test/WebpackConfig.js

```diff
@@ -546,7 +546,7 @@ describe('WebpackConfig object', () => {
 
         it('Calling with "includeNodeModules" option', () => {
             const config = createConfig();
-            config.configureBabel(() => {}, { include_node_modules: ['foo', 'bar'] });
+            config.configureBabel(() => {}, { includeNodeModules: ['foo', 'bar'] });
```

Comment:
should already be done in the existing version

Reply:
I remember keeping the `include_node_modules` check on purpose since we only deprecated that version. But maybe we should have had an additional check for the uppercase version.